python-backend/app/api/document_manager/models/document_manager.py

```diff
@@ -0,0 +1,30 @@
+from datetime import datetime
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+
+from ...utils.models_mixins import AuditMixin, Base
+
+
+class DocumentManager(AuditMixin, Base):
+    __tablename__ = 'document_manager'
+    document_manager_id = db.Column(db.Integer, primary_key=True, server_default=FetchedValue())
+    document_guid = db.Column(UUID(as_uuid=True), nullable=False)
+    full_storage_path = db.Column(db.String(150), nullable=True)
+    upload_date = db.Column(db.DateTime, nullable=False)
+    file_display_name = db.Column(db.String(40), nullable=False)
+    path_display_name = db.Column(db.String(150), nullable=True)
+
+    def __repr__(self):
+        return '<Document_manager %r>' % self.party_guid
+
+    def json(self):
+        return {
+            'document_manager_id' : str(self.document_manager_id),
```

Comment:
I don't believe we're putting a space before the colon in other dictionaries

Also, if this is just an integer then we shouldn't need to coerce it to a string to be valid JSON. Is there a reason why it should be a string?